src/get-started.md.njk

```diff
@@ -0,0 +1,95 @@
+---
+layout: layout-single-page-prose.njk
+---
+
+# Get started
+
+{% from "_example.njk" import example %}
+
+This guide explains how to create prototypes using the GOV.UK Design System and GOV.UK Prototype Kit.
+
+## Before you start
+
+To make prototypes you will need to download and install a new version of the GOV.UK Prototype Kit which has been built to work with the Design System while it’s in private beta.
+
+The installation process is the same as for the current version. You should:
+
+* [download the new version of the GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com/)
+* [follow these installation instructions](https://govuk-prototype-kit-beta.herokuapp.com/docs/install/introduction)
+
+The rest of the guide assumes that you are familar with the GOV.UK Prototype Kit.
+
+## Using the new Prototype Kit
+
+The new version of the Prototype Kit is identical to the one you may have used before except that it uses a new frontend framework called GOV.UK Frontend.
+
+This means that any code that you copy across from old prototypes will not display correctly. Instead, you should use the code provided in the Design System.
+
+## Styling content
+
+The Design System provides lots of new CSS classes for styling content, meaning you shouldn’t need to write your own Sass or CSS.
+
+The class names follow the [BEM naming convention](https://en.bem.info/methodology/quick-start/). This can look a bit daunting at first, but it makes for more robust code that’s easier to maintain.
+
+There are 2 ways to apply styles to content.
+
+
+### Using a prose scope wrapper
+
+To give basic HTML elements a default GOV.UK style, wrap them in a ‘govuk-prose-scope’ div, like this:
+
+<pre class="govuk-!-mb-r6"><code class="hljs html"><span class="hljs-tag">&lt;<span class="hljs-name">div</span> <span class="hljs-attr">class</span>=<span class="hljs-string">"govuk-prose-scope"</span>&gt;</span>
```

Comment:
It'd be good to have a way to include HTML without the example preview so that we don't need to do this, but think I'm happy for that to happen later.